app/db/migrations/20210209160716-index-streets-and-users.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addIndex('Streets', ['id'])
+    await queryInterface.addIndex('Streets', ['original_street_id'])
```

Comment:
We actually don't want to add an index to `original_street_id`. This is a pointer to a street that a new street was forked from, if any. Multiple streets can be copied from a single original street, so this field would not be unique.

Reply:
We might need to talk more about this so I understand, but I think I see what you're saying. However, we can still add an index even if the value isn't unique and that would likely still improve performance. 

Here is where that id is used in the code: https://github.com/streetmix/streetmix/blob/3a5d2dc503fc2afecf83512f0b8ce41f37545045/app/resources/v1/streets.js#L500

So as written, its still going to need to search through _all_ streets until it finds one that matches. If done sequentially it could be quite slow. 

Furthermore, I want to understand the non-uniqueness you're talking about here, because as written its just returning the first result of that ID. That seems pretty problematic if that ID isn't actually for the correct street for that user if there is more than one that might match?

Reply:
We talked about this in Discord - but noting for the future - that the where request is still on the `id` field and not the `original_street_id` field. So we won't index this for now, until we know why we need it.